### ISSUE TITLE
Alignment with other CCAMP YANG models needed

### ISSUE THREAD

**Issue by dieterbeller** [ietf-ccamp-wg/draft-ietf-ccamp-flexigrid-media-channel-yang#1]
The ietf-flexi-grid-media-channel YANG model shall make use of:

- ietf-layer0-types (https://github.com/ietf-ccamp-wg/draft-ietf-ccamp-layer0-types)
- ietf-layer0-types-ext (https://github.com/ietf-ccamp-wg/ietf-ccamp-layer0-types-ext/blob/master/ietf-layer0-types-ext.yang)

It should at least be aligned with:

- ietf-optical-impairment-topology (https://github.com/ietf-ccamp-wg/draft-ietf-ccamp-optical-impairment-topology-yang/blob/master/ietf-optical-impairment-topology.yang)

**Comment by danielkinguk** [comment]
for Daniel

**Comment by dperdices** [comment]
I mostly fixed all the errors of the module and now it is almost aligned with current version of layer0-types, layer0-types-ext and ietf-te-topology. Here is a summary:

-  Whenever a grouping of layer0-types was not found, we look for it in the current version. Most of them were just renamed so it was only a matter of finding the right one.
- Some of the old groupings are no longer in the layer0-types. FEC was in layer0-types-ext but term-type, wavelength assignment and flexi-grid-path-bandwidth were eliminated without clear equivalents. 
- Some TE-topology paths were also wrong. In most cases, it was easy to update them but, in some cases, I didn't find an equivalent node in the current TE-topo doc. In these cases, I comment out those sections.
- In layer-0-types, there is an error causing validation problems. In line 523, I think `must '. >= min-slot-width-factor'` should be `must '. >= ../min-slot-width-factor'`. If someone checks that I am right, we can report this to the authors.

 Any help or comment is welcome with the last three points! :)

**Comment by dperdices** [comment]
These are the validation errors of this WIP version;:
```
/tmp/tmpotj8m0oa/ietf-layer0-types-ext.yang:272: error: RFC 8407: 4.14: statement "container" must have a "description" substatement
/tmp/tmpotj8m0oa/ietf-flexi-grid-media-channel.yang:73: error: identity "term-type" not found in module "ietf-layer0-types"
/tmp/tmpotj8m0oa/ietf-flexi-grid-media-channel.yang:92: error: identity "wavelength-assignment" not found in module "ietf-layer0-types"
/tmp/tmpotj8m0oa/ietf-flexi-grid-media-channel.yang:119: error: grouping "flexi-grid-path-bandwidth" not found in module "ietf-layer0-types"
/tmp/tmpotj8m0oa/ietf-flexi-grid-media-channel.yang:127: error: grouping "flexi-grid-path-bandwidth" not found in module "ietf-layer0-types"
/tmp/tmpotj8m0oa/ietf-flexi-grid-media-channel.yang:137: error: grouping "flexi-grid-path-bandwidth" not found in module "ietf-layer0-types"
/tmp/tmpotj8m0oa/ietf-flexi-grid-media-channel.yang:148: error: grouping "flexi-grid-path-bandwidth" not found in module "ietf-layer0-types"
/tmp/tmpotj8m0oa/ietf-flexi-grid-media-channel.yang:158: error: grouping "flexi-grid-path-bandwidth" not found in module "ietf-layer0-types"
```

**Comment by italobusi** [comment]
Addressed in PR #13 

Pending merging of https://github.com/ietf-ccamp-wg/ietf-ccamp-layer0-types-ext/pull/17
